src/pages/academy/blog/index.tsx

```diff
@@ -69,6 +69,38 @@ const ArticlesPage = ({ data }) => {
         og_description:
             'Educational content at your fingertips – everything you need to know to start trading or upgrade your trading skills.',
     }
+
+    const [scrollPosition, setScrollPosition] = React.useState(0)
+    const handleScroll = () => {
+        const position = window.pageYOffset
+        setScrollPosition(position)
+    }
+
+    React.useEffect(() => {
```

Comment:
import `useEffect` from react 

```suggestion
   useEffect(() => {
```